go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/oklog/ulid
+
+require github.com/pborman/getopt v0.0.0-20170112200414-7148bc3a4c30
```

Comment:
This dependency is only needed for what's in `cmd`, not the library. What's the recommended approach in Go modules to deal with situations like this?

Reply:
because this is part of the source code, dependencies must be included in the main list of dependencies.

```
→ go mod tidy -v
→ cat go.mod 
module github.com/oklog/ulid

require github.com/pborman/getopt v0.0.0-20170112200414-7148bc3a4c30
```

Reply:
Although it is technically possible to have separate go.mod for e.g. subdirectories in a repo, [it is not recommended](https://twitter.com/FiloSottile/status/1096165600890707973). So I think this is the best outcome.